assembly/spdx-license-knowledge-base/pom.xml

```diff
@@ -73,6 +73,22 @@
             <artifactId>jsonld-java</artifactId>
```

Comment:
Sorry, that I come up with this comment so late, but I think we should also move the spdx license knowledgebase to the core workflow steps and get rid of another module.